src/client.cpp

```diff
@@ -40,6 +41,7 @@ using Request     = http::request<http::string_body>;
 
 static fs::path REPO_ROOT;
 static const fs::path OUINET_CONF_FILE = "ouinet-client.conf";
+static boost::posix_time::time_duration MAX_CACHED_AGE = boost::posix_time::hours(7*24);  // one week
```

Comment:
rather than hardcoding this i would pull this from a config ... do we have a config "thing" or are you just using '.h' files?

Reply:
@timball that is a hardcoded _default_ value. One can modify it using the [`--max-cache-age`](https://github.com/equalitie/ouinet/blob/acb582b4c12cc518156b99777bf62aeb0f9161d9/src/client.cpp#L445) command line argument or by adding it into [this config file](https://github.com/equalitie/ouinet/blob/acb582b4c12cc518156b99777bf62aeb0f9161d9/repos/injector/ouinet-injector.conf).

Reply:
oh snap! you all are the greatest! love you guys